group_vars/postgresql/production.yml

```diff
@@ -1,46 +1,43 @@
 ---
-postgres_version: "13"
+postgres_version: "15"
 db_clusteradmin_user: "postgres"
 db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
-replication_user_password: "{{ vault_replicant_password }}"
 postgresql_settings:
   - name: port
     value: 5432
   - name: listen_addresses
     value: "*"
 postgresql_ext_install_postgis: true
 postgresql_databases:
-  - name: replicant
   - name: postgres
 postgresql_users:
   - name: postgres
     password: "{{ vault_postgres_admin_password }}"
     db: "postgres"
     priv: "ALL"
     role_attr_flags: "SUPERUSER"
-  - name: replicant
-    password: "{{ vault_replicant_password }}"
-    db: "replicant"
-    priv: "ALL"
-    role_attr_flags: "SUPERUSER"
-postgres_leader: "{{ leader_db_ip }}"
-postgres_standby: "{{ standby_db_ip }}"
+  - name: datadog
```

Comment:
This seem like a strange addition?

Reply:
you need to create a datadog user to monitor postgresql.